### PR TITLE
Add 'make install' target

### PR DESCRIPTION
**Pull request by jpereira**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 This is a Packer plugin for installing Windows updates (akin to [rgl/vagrant-windows-update](https://github.com/rgl/vagrant-windows-update)).
 
-**NB** This was only tested with Packer 1.5.1 and Windows Server 2019.
+**NB** This was only tested with Packer 1.5.1 on Windows Server 2019, OSX Catalina and Ubuntu 20.04.
```
Comment (rgl) [inline README.md:9]: Isn't it called macOS instead of OSX?

File: README.md [hunk 2]
```diff
@@ -101,5 +101,11 @@ following snippet to your `~/.packerconfig` (or `%APPDATA%/packer.config`):
 }
 ```
 
+Or just place in `$HOME/.packer.d/plugins`
```
Comment (rgl) [inline README.md:104]: Use this phrase instead?

> Or install into `$HOME/.packer.d/plugins` with:

File: Makefile [hunk 3]
```diff
@@ -1,3 +1,4 @@
+UNAME_S := $(shell uname -s | tr '[:upper:]' '[:lower:]')
```
Comment (rgl) [inline Makefile:1]: Instead of using `uname`, this should use `go env` as:

```
GO_HOST_OS := $(shell go env GOHOSTOS)
```

Comment (jpereira) [reply Makefile]: yeap. done

File: README.md [hunk 4]
```diff
@@ -6,7 +6,7 @@
 
 This is a Packer plugin for installing Windows updates (akin to [rgl/vagrant-windows-update](https://github.com/rgl/vagrant-windows-update)).
 
-**NB** This was only tested with Packer 1.5.1 and Windows Server 2019.
+**NB** This was only tested with Packer 1.5.1 on Windows Server 2019, MacOSx Catalina and Ubuntu 20.04.
```
Comment (rgl) [inline README.md:9]:
```suggestion
**NB** This was only tested with Packer 1.5.1 on Windows Server 2019, macOS Catalina and Ubuntu 20.04.
```

File: README.md [hunk 5]
```diff
@@ -101,5 +101,11 @@ following snippet to your `~/.packerconfig` (or `%APPDATA%/packer.config`):
 }
 ```
 
+Or Or install into `$HOME/.packer.d/plugins with:`
```
Comment (rgl) [inline README.md:104]:
```suggestion
Or install into `$HOME/.packer.d/plugins` with:
```

**Comment by jpereira** [comment]
yep! make sense, updated.

```
[jpereira@jorge-sugarloaf:packer-provisioner-windows-update.git]$ make install
cd update && go run assets_generate.go
writing assets_vfsdata.go
mkdir -p build/darwin/
GOOS=darwin GOARCH=amd64 go build -v -o build/darwin/packer-provisioner-windows-update
github.com/rgl/packer-provisioner-windows-update/update
github.com/rgl/packer-provisioner-windows-update
mkdir -p /Users/jpereira/.packer.d/plugins
cp -f build/darwin/packer-provisioner-windows-update /Users/jpereira/.packer.d/plugins/packer-provisioner-windows-update
[jpereira@jorge-sugarloaf:packer-provisioner-windows-update.git]$
```

**Pull request merged**